client/web/src/components/series/list/SeriesList.tsx

```diff
@@ -38,39 +23,29 @@ export const SeriesList: React.FC<SeriesListProps> = () => {
   }
 
   return (
-    <TableContainer component={Paper}>
-      <Grid container justify="flex-end">
-        <Button 
-          variant="contained"
-          onClick={() => synchronize()}
-          startIcon={<SyncIcon />}>{t('Refresh')}
-        </Button>
-      </Grid>
-      <Table className={classes.table} aria-label="simple table">
-        <TableHead>
-          <TableRow>
-            <TableCell>seriesInstanceUID</TableCell>
-            <TableCell align="right">Series description</TableCell>
-            <TableCell align="right">Modality</TableCell>
-            <TableCell align="right">Number Of Instances</TableCell>
-            <TableCell align="right">Body part</TableCell>
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {list &&
-            list.map(({ id, seriesInstanceUID, seriesDescription, modality, numberOfSeriesRelatedInstances }) => (
-              <TableRow key={id}>
-                <TableCell component="th" scope="row">
-                  {seriesInstanceUID}
-                </TableCell>
-                <TableCell align="right">{seriesDescription}</TableCell>
-                <TableCell align="right">{modality}</TableCell>
-                <TableCell align="right">{numberOfSeriesRelatedInstances}</TableCell>
-                <TableCell align="right">---</TableCell>
-              </TableRow>
-            ))}
-        </TableBody>
-      </Table>
-    </TableContainer>
+    <div style={{ maxWidth: '100%' }}>
+      <MaterialTable
+        title="Series list"
+        columns={[
+          { title: 'Series instance UID', field: 'seriesInstanceUID' },
```

Comment:
you should use the translation system about title

Reply:
Done